library/ssl_misc.h

```diff
@@ -2088,6 +2088,8 @@ int mbedtls_ssl_tls13_write_early_data_ext( mbedtls_ssl_context *ssl,
 static inline uint8_t mbedtls_ssl_tls13_early_data_state(
                           mbedtls_ssl_context *ssl )
 {
+    if( ssl->handshake == NULL )
```

Comment:
So it seems that my analysis was wrong.

Reply:
That's my mistake. I should keep handshake check.